src/netguard/include/fd_util.h

```diff
@@ -0,0 +1,11 @@
+//
+// Created by Aitor Viana on 25/05/2023.
+//
+
+#ifndef NETGUARD_FD_UTIL_H
+#define NETGUARD_FD_UTIL_H
+
+int is_readable(int fd);
+int is_writable(int fd);
```

Comment:
Extract them to new `fd_util` file to avoid pulling other types and dependencies that are in `utils` file that we dont need for netp